pydynamo/storage/disk/sstable.py

```diff
@@ -0,0 +1,95 @@
+from ..store import Store
+from ..memory.inmemorystore import InMemoryStore
+from ..error import StorageException
+from ..error import ErrorType
+from datetime import datetime
+from .sstableiterator import SStableIterator
+from struct import pack
+
+
+class SSTable(Store):
+    def __init__(self, store_name: str, store_id: int,
+                 mem_table: InMemoryStore, percentage: float) -> None:
```

Comment:
percentage is not a good name, consider something like "index_ratio" instead? also please add comment for what it means.